test/extension/commandExecutor.test.ts

```diff
@@ -125,5 +130,65 @@ suite("commandExecutor", function() {
                     assert.fail(null, null, "react-native command was not expected to fail");
                 });
         });
+
+        test("getReactNativeVersion should return verson string if there is react-native package in node_modules", (done: MochaDone) => {
+            let commandExecutor: CommandExecutor = new CommandExecutor(path.join(__dirname, "..", "resources", "sampleReactNative022Project"));
+
+            Q({})
+            .then(function() {
+                return commandExecutor.getReactNativeVersion();
+            })
+            .then(function(version) {
+                assert.equal(version, "^0.22.2");
+            }).done(() => done(), done);
+        });
+
+        suite("ReactNativeClIApproaches", function () {
+
+            const sampleReactNative061ProjectDir = path.join(__dirname, "..", "resources", "sampleReactNative061Project");
+            let uri: vscode.Uri;
+            const correctRNGlobalCLINameContent: any = {
+                "react-native-tools.reactNativeGlobalCommandName": "react-native",
+            };
+
+            const incorrectRNGlobalCLINameContent: any = {
+                "react-native-tools.reactNativeGlobalCommandName": "incorrect",
+            };
+
+            setup(() => {
+                cp.execSync("npm i", { cwd: sampleReactNative061ProjectDir });
```

Comment:
I'm not sure if we need to use `npm i` here as we can create fake folders tree if we need.